src/engraving/dom/chordlist.cpp

```diff
@@ -1838,7 +1838,9 @@ bool ChordList::read(const muse::io::path_t& appDataPath, const String& name)
     }
     File f(path);
     if (!f.open(IODevice::ReadOnly)) {
-        LOGE() << "Cannot open chord description: " << f.filePath();
+        if (!MScore::testMode) {
+            LOGE() << "Cannot open chord description: " << f.filePath();
```

Comment:
I actually don't think it's an error? If this ChordList is stored in a separate file from the mscx, but unit tests only use mscx, then it's expected for it not to be found. Or at least that how I interpreted this

Reply:
Indeed, it's as Jojo says: this file is part of a MuseScore Studio installation, but during unit tests it is not found, because those don't use a MuseScore Studio installation but only a standalone test executable.
An attempt to fix it was in https://github.com/musescore/MuseScore/pull/22508/files#diff-c0e02094285137de398298580648dbe1f158ad260fda44b19b7089bc2939eb2c, but either that did not fully fix it, or it was broken again. 
So the solution would be to tell the unit tests where they can find this file, for example by injecting the path to the file in the source repo, or to the "installed" file (because I do think we perform an install, but we just don't use it from there).